packages/core/integration-tests/test/incremental-bundling.js

```diff
@@ -712,21 +713,33 @@ console.log('index.js');`,
 
     it('changing bundler options', async () => {
       let subscription;
-      let fixture = path.join(__dirname, '/integration/incremental-bundling');
       try {
-        let b = bundler(path.join(fixture, 'index.js'), {
+        await fsFixture(overlayFS, __dirname)`
```

Comment:
I migrated this test as it was incorrectly sending file `create` events rather than `update` events causing the config to not invalidate. This is a bug when using overlayFS with a normal read FS but an in memory write FS.